File: src/ast/ConversionOperatorType.java
package ast;

import java.io.StringWriter;

public class ConversionOperatorType extends ParentNode {
    public ConversionOperatorType(BaseNode child) {
        super(NodeType.ConversionOperatorType, child);
    }

    @Override
    public void PrintLeft(StringWriter writer) {
        writer.write("operator ");
        child.Print(writer);
    }
}

File: src/de/fearlesstobi/demangler/ast/ConversionOperatorType.kt
package de.fearlesstobi.demangler.ast

import java.io.StringWriter

class ConversionOperatorType(child: BaseNode?) : ParentNode(NodeType.ConversionOperatorType, child) {
    public override fun printLeft(writer: StringWriter) {
        writer.write("operator ")
        child!!.print(writer)
    }
}